### ISSUE TITLE
Wrong version of Dart SDK being selected

### ISSUE THREAD

**Issue by AylanBoscarino** [steebchen/flutter#5]
Trying the example yaml on this repo's readme ends up failing the install dependencies steps with the following message:

> Because exame depends on json_serializable 3.2.5 which requires SDK version >=2.6.0 <3.0.0, version solving failed.
> pub get failed (1)

My pubspect.yml file points to sdk ">=2.3.0 <3.0.0" and when running the pub get on the action it sets the version to 2.5.0

Everything runs normal on a local machine.

My action file:

```
name: Dart CI

on:
  push:
    branches: [ master, development ]
  pull_request:
    branches: [ master, development ]

jobs:
  build:
    name: install dependencies
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v1

    - name: install dependencies
      uses: steebchen/flutter@v1.0.0
      with:
        args: pub get

    - name: run tests
      uses: steebchen/flutter@v1.0.0
      with:
        args: test

    - name: build apk
      uses: steebchen/flutter@v1.0.0
      with:
        args: build apk --release
```

The complete error message: 

![image](https://user-images.githubusercontent.com/30734219/77225742-ae0f1080-6b50-11ea-912f-ce9c624b1097.png)

Am I doing something wrong?

**Comment by sthbryan** [comment]
i have the same problem with firestore

**Comment by Shiba-Kar** [comment]
facing same issue

**Comment by caiogranero** [comment]
Same problem here

**Comment by steebchen** [comment]
This repository currently does not have a way to choose specific flutter sdk versions. I'd be happy if anyone could send a PR or help contributing. I don't know when I might get to this.

**Comment by martinloesethjensen** [comment]
@steebchen the problem should now have been resolved as the flutter version have been bumped to `v1.12.13+hotfix.9` in the [`Dockerfile`](https://github.com/steebchen/flutter/blob/1db2c6f0efa2dd25c8e62e908350e4efe306c6fb/Dockerfile#L1) you use from @gableroux 😄 

Here is the [PR](https://github.com/GabLeRoux/docker-flutter/pull/1) from the repo 🚀   

Have a great day and hope this problem has been resolved 😄

**Comment by martinloesethjensen** [comment]
@AylanBoscarino could you verify if this is still occurring 👍 
No update of the version should be required 🤔

**Comment by steebchen** [comment]
Released as [1.1.0](https://github.com/steebchen/flutter/releases/tag/v1.1.0), use `steebchen/flutter@1.1.0` in the GitHub action.
